### ISSUE TITLE
Peers are listed as disconnected in kademlia but they are not

### ISSUE THREAD

**Issue by janos** [ethersphere/bee#234]
Some of the peers are listed in kademlia topology response as disconnected even they are not according to debug api /peers response, log messages and pingpong functioning properly.

This issue is the a followup on https://github.com/ethersphere/bee/issues/219 with the difference that the consequence different but manifestation is similar.

**Comment by vandot** [comment]
Trying to implement node discovery inside local network using multicast I get this issue every time. Implementation is that node is started and it sends discovery packet, every other running node that receives that message connects to that node.

Interesting thing is that in cluster of 5 nodes node `0` (started first so in the end it initiates connection to every node started after) has all 4 nodes listed as disconnected, node `1` has one connected node (node 0) and 3 disconnected, node `2` 2 connected (node 0 and 1) and two disconnected, up to last node that has all nodes listed as connected (all other nodes initiated connection to it)

TL;DR if the node initiate connection to other node it will set that node as disconnected in `/topology`, node that was connected to will keep initiator as connected.

fullconnectivity and pingpong tests passes and no log messages about disconnecting nodes
